module2.js

```diff
@@ -36,8 +47,8 @@ function question1(personObj) {
 */
 
 function question2(carObj) {
-  // TODO
-  return /* ??? */
+  carObj.price = 3000;
+  return 'The price is ' + carObj.price;
```

Comment:
This passes the test, but doesn't actually meet the requirements of the description! Reread the comments above the function definition and see if you can work out how else you could solve the problem.